### PR TITLE
Wst24365888 proposal submission

### PR DESCRIPTION
**Pull request by wst24365888**
---
A good PR should include the following parts:

- [x] A clear title
- [x] A meaningful message for PR, as well as its commits
- [x] from your specific branch (***not master***) merging to teacher's `HW` branch (***not master***, again)
- [x] excluding any irrelevant files, such as binaries, text files, or dot files
- [x] passing tests/CI

### CODE REVIEW

File: project/wst24365888/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  - IoT
+  - `RPC` frameworks like [thrift](https://github.com/facebook/fbthrift), [grpc](https://github.com/grpc/grpc) (they use `protobuf (varint)` instead)
+
+## System Architecture
+
+![System Architecture](./assets/system-architecture.png)
+
+## API Description
+
+- `Python` API
```
Comment (yungyuc) [inline project/wst24365888/README.md:31]: Do you plan to support Python byte and str types?

**Review by yungyuc — APPROVED** [review]
LGTM.  It will be good to include the github link in the propose.  Let me know when you want this to get merged.

File: project/wst24365888/README.md (hunk 1 above)
Comment (wst24365888) [reply project/wst24365888/README.md:33]: If I still have time after finish current `Python` APIs in proposal, I will. :)

**Comment by wst24365888** [comment]
> LGTM. It will be good to include the github link in the propose. Let me know when you want this to get merged.

I think it can be merged. Thanks for the review!

File: project/wst24365888/README.md (hunk 1 above)
Comment (yungyuc) [reply project/wst24365888/README.md:33]: Fair enough.

**Pull request merged**